subsys/mgmt/hawkbit/hawkbit_config_data.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 Advanced Climate Systems
+ *
+ * SPDX-License-Identiier: Apache-2.0
+ */
+
+#ifndef __HAWKBIT_CONFIG_DATA_H__
+#define __HAWKBIT_CONFIG_DATA_H__
+
+#include <data/json.h>
+#include <autoconf.h>
```

Comment:
I think `autoconf.h` should not be included

Reply:
Should be addressed now